README.md

```diff
@@ -94,15 +94,46 @@ You can now use the flag `--javascript-engine=em` to specify EmberScript
 assets in your generators, but all of the generators will default to
 using an EmberScript variant first.
 
+## For ES6 Modules support
+
+Ember.js recommends using ES6 module syntax.
+It is supported by [es6_module_transpiler](https://github.com/dockyard/es6_module_transpiler-rails).
+
+Run the bootstrap generator with an extra flag:
+
+``` sh
+rails g ember:bootstrap --javascript-engine=es6
+```
+
 Note:
 
-Ember-rails include some flags options for bootstrap generator:
+To use ES6 module in your application, the following configuration is required:
+
+`my_app.js.es6`
+``` javascript
+//= require ember-rails/application
+// And require more modules...
+
+import Ember from 'ember-rails/application';
+
+var App = Ember.Application.extend();
 
+App.craete();
 ```
---ember-path or -d # custom ember path
---skip-git or -g # skip git keeps
---javascript-engine  # engine for javascript (js, coffee or em)
---app-name or -n # custom ember app name
+
+`require ember-rails/application` is important.
+It provides customized Ember application to resolve dependencies from ES6 modules.
+
+`application.js`
+``` javascript
+//= require jquery
+//= require handlebars
+//= require ember
+//= require ember-data
+//= require ./my_app
```

Comment:
lets try to align this with ember-cli's conventions, all files/folders in the app are dasherized